src/neo/SmartContract/Native/GasToken.cs

```diff
@@ -15,7 +15,7 @@ internal GasToken()
         internal override void Initialize(ApplicationEngine engine)
         {
             UInt160 account = Contract.GetBFTAddress(engine.ProtocolSettings.StandbyValidators);
-            Mint(engine, account, 30_000_000 * Factor, false);
+            _ = Mint(engine, account, 30_000_000 * Factor, false);
```

Comment:
this call should be awaited. Since it's void return, we need a try/catch block as well